File: 151003/Alekseeva/TaskREST/TaskREST/Dto/Response/TweetResponseTo.cs
using System.Text.Json.Serialization;

namespace TaskREST.Dto.Response;

public record TweetResponseTo(
    [property: JsonPropertyName("id")] long Id,
    [property: JsonPropertyName("creatorId")]
    long CreatorId,
    [property: JsonPropertyName("title")] string Title,
    [property: JsonPropertyName("content")]
    string Content
);

File: 151001/Pastukhou/RV2/src/main/java/com/bsuir/kirillpastukhou/domain/response/TweetResponseTo.java
package com.bsuir.kirillpastukhou.domain.response;

import java.time.LocalDateTime;
import java.util.List;

public record TweetResponseTo(
        Long id,
        Long creatorId,
        String title,
        String content,
        LocalDateTime created,
        LocalDateTime modified, //   List<TagResponseTo> stickerList
        List<MessageResponseTo> messageList) {
}
